pkg/router/route_group.go

```diff
@@ -418,7 +420,7 @@ func (rg *RouteGroup) close(code routing.CloseCode) error {
 		rg.closeDone.Add(len(rg.tps))
 	}
 
-	rg.broadcastClosePackets(code)
+	go rg.broadcastClosePackets(code)
```

Comment:
I feel like we should use `rg.writePacketAsync` within `rg.broadcastClosePackets`, instead of calling `rg.broadcastClosePackets` in a go routine.